packages/story-editor/src/components/form/stackable/index.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+
+const StackableContainer = styled.div(
+  ({ locked, styleOverride }) => css`
+    display: flex;
+    ${locked ? 'max-width: 106px' : ''}
+    ${styleOverride};
+
+    & > div {
+      border-radius: 0;
+
+      &:first-of-type > div {
+        border-radius: 4px 0 0 4px;
+      }
+      &:last-of-type > div {
+        border-radius: 0 4px 4px 0;
+      }
```

Comment:
I feel like these should go to the generalized StackableInput as well, which makes the CSS faster as CSS rules are evaluated right-to-left.

The right-most part of this rule is "div" matching thousands of elements, so the CSS parser has to go one step further before discarding this rule for almost all elements.

With styled-components we're almost exclusively using single-item CSS selectors for speed reasons, because we can and only multi-item selectors when unavoidable. And when we do use multi-level, we make them as specific as possible. Here you could have made it more specific if you had a reference to that other div and did:

```
& > ${StackableInput} {
```

Styled-components knowns how to compose those and replace the component with its generated class name, which is pretty neat.